lib/config/config.js

```diff
@@ -200,11 +200,17 @@ module.exports = function (fs, path, url, convict) {
         env: 'IP_BLOCKLIST_LOGONLY'
       },
       lists: {
-        doc: 'A array of ip blocklist file paths',
+        doc: 'An array of ip blocklist file paths',
         format: Array,
         default: [],
         env: 'IP_BLOCKLIST_FILES'
       },
+      listsOverrideRules: {
+        doc: 'An array of true/false values that map directly the the lists property and determines if that specific list should be blocked',
+        format: Array,
+        default: [],
+        env: 'IP_BLOCKLIST_FILES_BLOCKING'
+      },
```

Comment:
Your approach seems much more reasonable and cleaner. I will update accordingly.